crates/router/src/configs/settings.rs

```diff
@@ -941,6 +945,20 @@ pub struct ServerTls {
     pub certificate: PathBuf,
 }
 
+#[cfg(feature = "v2")]
+#[derive(Debug, Clone, Deserialize, PartialEq, Eq)]
+pub struct CellInformation {
+    pub id: common_utils::id_type::CellId,
+}
+
+#[cfg(feature = "v2")]
+impl Default for CellInformation {
+    fn default() -> Self {
+        let cell_id = common_utils::id_type::CellId::from_str("abcde")
+            .expect("Failed to create a default for Cell Id");
```

Comment:
1. Nit: Can we have the default value to something along the lines of `default_cell_id`? To be more clear that it's a default...
2. Add `#[allow(clippy::expect_used)]` above this statement.